client/utils.py

```diff
@@ -1,20 +1,35 @@
-import requests
+import configparser
 import os
-from openai import OpenAI
-import pyperclip
 import sys
+from dataclasses import dataclass
+from typing import Self
```

Comment:
[New in version 3.11](https://docs.python.org/3/library/typing.html#typing.Self)